src/mui-theme-provider.jsx

```diff
@@ -0,0 +1,23 @@
+import {Component, PropTypes} from 'react';
+
+class ThemeProvider extends Component {
+  getChildContext() {
+    return {
+      muiTheme: this.props.muiTheme,
+    };
+  }
+  render() {
+    return this.props.children;
+  }
+}
+
+ThemeProvider.propTypes = {
+  children: PropTypes.element,
+  muiTheme: PropTypes.object,
+};
+
+ThemeProvider.childContextTypes = {
```

Comment:
if you can't find anything just disable the faulted rules for this file. (with a tiny comment saying why) Thanks a lot.